newlib/testsuite/newlib.iconv/meson.build

```diff
@@ -38,7 +38,7 @@ tests = ['iconvjp', 'iconvnm', 'iconvru' ]
 iconv_data_link = custom_target('iconv_data link',
 				install: false,
 				output: 'iconv_data',
-				command: ['ln', '-srf', '@SOURCE_DIR@' / 'newlib/libc/iconv/ccs/binary', '@OUTPUT@'])
```

Comment:
This makes tests fail when built with external iconv files as with this configuration:
```
../../scripts/do-native-configure -Dnewlib-iconv-external-ccs=true
```

Reply:
Hmm right, should we maybe extract this into a separate shell script? That way we can try some fallback solutions if ln fails.

Reply:
I think all that is required is to figure out what paths to pass to ln without the -r option. I suppose a shell script could compute them?

Reply:
ok, I've pushed a ln fix to the non-linux-ln branch; that uses an absolute path instead of a relative path, which means the -r flag isn't needed anymore. Feel free to test that on your machines; if it passes the CI tests, I'll merge it to main.

Reply:
Your `ln` patch works on my machine, thanks @keith-packard !